client/go/cmd/config_test.go

```diff
@@ -101,10 +101,17 @@ func TestUseAPIKey(t *testing.T) {
 	}
 }`
 	withEnv("VESPA_CLI_CLOUD_SYSTEM", "public", func() {
+		ci, ok := os.LookupEnv("CI")
```

Comment:
Hehe :)

Reply:
🙈 
Commands need some refactoring so that the command itself and dependencies are contained in a struct. Tests can then pass mocks for things such as HTTP client and process environment into this struct.